lib/ants.rb

```diff
@@ -1,5 +1,6 @@
 require "mongoid"
 require "mongoid_slug"
+require 'mongoid_search'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.